extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -203,30 +205,70 @@ describes.realWin('amp-consent', {
           },
         },
       };
-      const consentElement = doc.createElement('amp-consent');
+      consentElement = doc.createElement('amp-consent');
       consentElement.setAttribute('id', 'amp-consent');
       consentElement.setAttribute('layout', 'nodisplay');
-      const scriptElement = doc.createElement('script');
+      scriptElement = doc.createElement('script');
       scriptElement.setAttribute('type', 'application/json');
       scriptElement.textContent = JSON.stringify(defaultConfig);
-      consentElement.appendChild(scriptElement);
       doc.body.appendChild(consentElement);
-      ampConsent = new AmpConsent(consentElement);
     });
 
     it('in geo group', function* () {
+      consentElement.appendChild(scriptElement);
+      ampConsent = new AmpConsent(consentElement);
       ISOCountryGroups = ['unknown', 'testGroup'];
       ampConsent.buildCallback();
       yield macroTask();
       expect(ampConsent.consentRequired_['ABC']).to.equal(true);
     });
 
     it('not in geo group', function* () {
+      consentElement.appendChild(scriptElement);
+      ampConsent = new AmpConsent(consentElement);
       ISOCountryGroups = ['unknown'];
       ampConsent.buildCallback();
       yield macroTask();
       expect(ampConsent.consentRequired_['ABC']).to.equal(false);
     });
+
+    it('promptIfUnknow override geo', function* () {
```

Comment:
can we make a 3rd case that promptIfUnknown=false should also override geo

Reply:
added